src/status_im2/contexts/communities/menus/community_options/__tests__/community_options_component_spec.cljs

```diff
@@ -0,0 +1,201 @@
+
+(ns status-im2.contexts.communities.menus.community-options.--tests--.community-options-component-spec
+  (:require ["@testing-library/react-native" :as rtl]
+            [re-frame.core :as re-frame]
+            [status-im2.contexts.communities.menus.community-options.view :as options]
+            [reagent.core :as reagent]))
+
+
+(defn setup-sub [opts]
```

Comment:
@ilmotta WDYT of this for tests in status_im2?

Reply:
@J-Son89 I'm afraid Jest is not automatically cleaning up registered subscriptions in re-frame's global registry. This is not a problem right now because we don't have many test competing for the same subscriptions, but I believe this solution won't work soon, because subscriptions registered during tests will have side-effects in other tests. At the very least I would expect some tests to "randomly" fail due to one test having registered a subscription that affects another one. 

Re-frame's registry is global, so one of the main features of the library https://github.com/day8/re-frame-test is to help clean up state between tests. You know we use this lib for writing integration tests and some event handler tests.

We either use some of the features provided by that lib to clean up the registry or we do it ourselves, like I did with the macro `status-im.test-helpers/deftest-sub`.

---

I think what you wrote here is fine for now, but someone will eventually be affected by the problem I described and we'll need to find a reliable and easy solution to setup subscriptions and clean them up between component tests.